.github/workflows/cd.main.yml

```diff
@@ -19,6 +19,6 @@ jobs:
 
           for track in */; do
               pushd $track
-              docker run -e INSTRUQT_TOKEN=${{ env.INSTRUQT_TOKEN }} --workdir="/track" --mount type=bind,source="$(pwd)",target=/track instruqt/cli track push
+              docker run -e INSTRUQT_TOKEN=${{ env.INSTRUQT_TOKEN }} --workdir="/track" --mount type=bind,source="$(pwd)",target=/track instruqt/cli track push --force
```

Comment:
Instead of calling `docker run` here we can run this entire job inside the Instruqt CLI container: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idcontainer.

Reply:
I tried this a while ago, and I think it's not working properly. It _may_ be that we're using a distroless container for the CLI, and we're missing some tooling.

Here is the error, you can check the changes in the workflow file for context:

https://github.com/instruqt/templates/runs/4865964838?check_suite_focus=true

Reply:
Ah that's a shame, thanks for the investigation.